src/components/NcAppSettingsDialog/NcAppSettingsDialog.vue

```diff
@@ -395,6 +395,9 @@ export default {
 
 <style lang="scss" scoped>
 .app-settings {
+	&:deep(.dialog) {
```

Comment:
I do not think we should add this here, but rather directly on the NcDialog component